static/postgres/v5-configuration/configuration.json

```diff
@@ -4429,5 +4429,6 @@
       "varchar": "string"
     }
   },
-  "mutationsVersion": "v2"
+  "mutationsVersion": "v2",
+  "mutationsPrefix": ""
```

Comment:
Before we added this, all the Postgres tests passed as before, after this, we had to go fix lots of things. Which means it works, which is good.